sympy/stats/error_prop.py

```diff
@@ -0,0 +1,74 @@
+"""Tools for arithmetic error propogation."""
+from __future__ import print_function, division
+from itertools import repeat
+
+from sympy import Symbol, symbols, Add, Mul, simplify, Pow, exp
```

Comment:
Indeed, it needs to be a tool that integrates into all editors, as opposed to a tool that emulates all editors.